model/src/main/java/org/bonitasoft/web/designer/model/MigrationStatusReport.java

```diff
@@ -16,13 +16,15 @@
  */
 package org.bonitasoft.web.designer.model;
 
-import com.fasterxml.jackson.annotation.JsonIgnore;
-import com.fasterxml.jackson.annotation.JsonView;
-
-public class MigrationStatusReport {
-
-    private boolean compatible;
-    private boolean migration;
+/**
+ * Report for artifact status
+ *
+ * @deprecated Not for public use.
+ * @candidate for removal in a future release.
+ *            This class don't have an explicit name. Replaced by {@link #ArtifactStatusReport}
+ */
+@Deprecated
+public class MigrationStatusReport extends ArtifactStatusReport {
 
     public MigrationStatusReport(boolean compatible, boolean migration) {
```

Comment:
Use super constructors and define `compatible` & `migration` as private in super class